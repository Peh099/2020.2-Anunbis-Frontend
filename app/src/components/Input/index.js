import React from 'react';
import { InputStyle } from './styles';

export default function Input(props) {
  return (
    <InputStyle
      {...props}
      type={props.type}
      placeholder={props.text}
      value={props.value}
      name={props.name}
      ref={props.register}
      autoComplete="off"
      onKeyPress={props.onkeydown}
      step={props.step}
      width={props.width}
    />
  );
}
