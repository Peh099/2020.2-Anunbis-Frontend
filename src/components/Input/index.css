.Input input, select{
    border: none;
    border-bottom: 1px solid #000000;
    background-color:#FFD54F;
    margin: 15px 0;
    max-width: 500px;
    max-height: 400px;
    align-self: right;
    padding-right: 75px;
}