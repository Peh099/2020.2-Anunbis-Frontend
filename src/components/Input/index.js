import React from "react";

export default function Input(props) {

    return (
        <div>
            <input type={props.type} placeholder={props.text} value={props.value} name={props.name} ref={props.refe}>
            </input>
        </div>
    );
}