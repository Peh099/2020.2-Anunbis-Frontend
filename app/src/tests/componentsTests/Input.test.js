import renderer from 'react-test-renderer';
import Input from '../../components/Input';

describe('Snapshot Input component', () => {
    it('matches the snapshot', () => {
        const tree = renderer.create(<Input />).toJSON()
    
    expect(tree).toMatchSnapshot()
    })
})