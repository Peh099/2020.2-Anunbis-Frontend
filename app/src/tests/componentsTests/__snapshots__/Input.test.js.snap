// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Input component matches the snapshot 1`] = `
<input
  className="sc-bdnxRM dXJtIu"
/>
`;
